demo/push-to-github-pages.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
Is this file marked as executable?  GitHub doesn't show anything.

Reply:
Yes, it is executable.  Silly github review tools.  :-)